Jenkinsfile

```diff
@@ -27,6 +27,12 @@ node('docker') {
         }
     }
 
+    stage('Source code linter') {
```

Comment:
This won't work since we don't you Jenkinsfile anymore. You have to update Makefile,